orcestra_book/orcestra.md

```diff
@@ -4,23 +4,76 @@
 
 The overarching objective of ORCESTRA is to better understand the physical mechanisms that organize tropical convection at the mesoscale, including the interaction of convective organization with tropical waves and air-sea interaction, and the impact of convective organization on climate and the Earth’s radiation budget and processes of tropical cyclogenesis. In addition to advancing understanding of tropical meteorology and atmospheric processes, ORCESTRA observations will to help calibrate and validate satellite remote sensing (especially EarthCARE) and a new generation of global ocean-eddy and storm-resolving climate models.
 
-ORCESTRA is an international initiative that combines different sub-campaigns:
-* [BOW-TIE - Beobachtung von Ozean und Wolken – Das Trans ITCZ Experiment](bowtie.md)
-* [CELLO - Cloud and EarthCARE caL/vaL Observations](cello.md)
-* [CLARINET - CLoud and Aerosol Remote sensing for EarThcare](clarinet.md)
-* [MAESTRO - Mesoscale organisation of tropical convection](maestro.md)
-* [PERCUSION - Persistent EarthCare underflight studies of the ITCZ and organized convection](percusion.md)
-* [PICCOLO - Process investigation of clouds and convective organization over the tropical ocean](piccolo.md)
-* [SCORE - Sub-Cloud Observations of Rain Evaporation](score.md)
-* [STRINQS - Soundings and TuRbulent eddy measurements in the ITCZ with a Network of QuadcopterS](strinqs.md)
-
-## Date and location
-ORCESTRA will happen in August and September 2024 in the tropical Atlantic region.
-
-```{figure} /figures/campaign_overview_scetch.jpeg
-:alt: ORCESTRA campaign overview with the main measurement platforms, preliminary flight and ship tracks plotted above the total water vapor field (ERA5) on 15th of August 2001 indicating the location of the ITCZ.
-:width: 800px
-:align: center
-
-ORCESTRA campaign overview with the main measurement platforms, as well as preliminary flight and ship tracks plotted above the total water vapor field (ERA5) on 15th of August 2001 indicating the location of the ITCZ.
+## Sub-campaigns in ORCESTRA
+
+ORCESTRA unites several sub-campaigns under its roof. The detailed scientific objectives of the sub-campaigns can be found on subpages respectively.
+
+````{grid} 4
+
+```{grid-item-card} 
+:img-top: logos/01_BOWTIE.svg
+:img-alt: BOWTIE
+:link: https://orcestra-campaign.org/bowtie.html
```

Comment:
Internal links should be relative, as absolute links might lead to different versions and more easily become outdated.
```suggestion
:link: bowtie
:link-type: doc
```

The same applies to the following cargs.